package.json

```diff
@@ -23,16 +26,29 @@
     "eslint-plugin-react": "^7.27.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "html-webpack-plugin": "^5.5.0",
+    "jest": "^28.1.3",
+    "jest-environment-jsdom": "^28.1.3",
+    "jest-puppeteer": "^6.1.1",
+    "mkdirp": "^1.0.4",
     "nps": "^5.10.0",
     "prettier": "^2.4.1",
+    "puppeteer": "^16.0.0",
     "raw-loader": "^4.0.0",
     "rimraf": "^3.0.0",
+    "serve": "^14.0.1",
+    "ts-jest": "^28.0.7",
     "typescript": "^4.7.4",
+    "wait-on": "^6.0.1",
     "webpack": "^5.68.0",
     "webpack-cli": "^4.10.0",
     "webpack-dev-server": "^4.7.4",
     "wireit": "^0.7.1"
   },
+  "pnpm": {
+    "neverBuiltDependencies": [
```

Comment:
This prevents puppeteer from trying run husky (which isn't even a prod dep) on installation. We then separately have chromium installation instructions.